### PR TITLE
Feature/scripts and documentation

### PR DESCRIPTION
**Pull request by sapieneptus**
# Motivation

This PR changes the name of the project and adds it to `Carthage` and `Cocoapods` as well as providing some initial documentation for how to build/install/use.

### CODE REVIEW

**Comment by krukow** [comment]
I haven't tested it but the docs look good 👍

**Comment by sapieneptus** [comment]
@topgenorth Feel free to give some feedback on the [docs](https://github.com/xamarinhq/test-cloud-xcuitest-extensions/blob/bd88cf03cf8fda5b638a0ad98e401ff8e59370d7/README.md) 

CC: @jraczak

File: README.md [hunk 1]
```diff
@@ -1,2 +1,144 @@
-# test-cloud-xcuitest-extensions
-An extension library for to provided added value when running XCUITest tests in Visual Studio Mobile Center and Xamarin Test Cloud
+# VSMobileCenterExtensions
+An extension library for to provided added value when running XCUITest tests in Visual Studio Mobile Center and Xamarin Test Cloud.
```
Comment (topgenorth) [inline README.md]: "...for to provided added value..." - This doesn't make sense.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,144 @@
...
+
+- [Installation](#installation)
+  - [Cocoapods](#cocoapods)
+  - [Carthage](#carthage)
+  - [Build from source](#building-from-source)
+- [Usage](#usage)
+  - [Objective-C](#objective-c)
+  - [Swift](#swift)
+
+# Installation
```
Comment (topgenorth) [inline README.md]: I'd have a requirements section. For example, what version of Ruby? What gems should I have installed?I may need homebrew.  

I'd be explicit that command line familiarity is expected. 

Also, what versions of iOS and Xcode do I need as a minimum (or state "These instructions were tested on iOS X.Y and Xcode version X.Y Yes, this should be obvious, but newcomers will find this guide and might not be that savvy")

File: README.md (hunk 1 above)
Comment (krukow) [inline README.md]: ... by allowing calls to the `label` which provides screenshots in test reports in VS Mobile Center and Xamarin Test Cloud ... Feel free to re-word :)

**Comment by sapieneptus** [comment]
@topgenorth Thanks for the feedback. 

Made a few updates, I'll leave the PR open for the day in case anyone has more to add.

**Pull request merged**